src/main/java/com/zendesk/maxwell/MaxwellConfig.java

```diff
@@ -366,6 +371,10 @@ private void setup(OptionSet options, Properties properties) {
 		this.redisDatabase		= Integer.parseInt(fetchOption("redis_database", options, properties, "0"));
 		this.redisPubChannel	= fetchOption("redis_pub_channel", options, properties, "maxwell");
 
+		if (this.maxwellMysql.sslMode == null) {
+			this.maxwellMysql.sslMode = SSLMode.DISABLED;
+		}
+
```

Comment:
maybe just set this as the default in the `MawellMysqlConfig` initializer?

Reply:
the reason I was defaulting to null was so we could differentiate between not set and disabled, since replication and schema SSLMode's need to be overridden only if they're not set

Reply:
I see.  Ok, then that's ok for now.  I may clean this up a bit later by maybe unifying some of this stuff under something like `MaxwellMysqlConfig#clone(replicationHost, replicationPassword, ...)` which can be smart about inheriting stuff.

Reply:
that sounds great. yeah I don't like the null checks, there's definitely a better way of doing it